ui/main/src/app/modules/core/application-loading/login/login.component.ts

```diff
@@ -32,7 +32,7 @@ export class LoginComponent implements OnInit {
             .pipe(filter((m) => m != null && m.level === MessageLevel.ERROR))
             .subscribe((m) => (this.loginMessage = m));
         this.userForm = new FormGroup({
-            identifier: new FormControl(''),
+            identifier: new FormControl('', [Validators.minLength(1)]),
```

Comment:
this component is for login , the test should not be done here but when adding new user in ui/main/src/app/modules/admin/components/editmodal/users/edit-user-modal.component.ts